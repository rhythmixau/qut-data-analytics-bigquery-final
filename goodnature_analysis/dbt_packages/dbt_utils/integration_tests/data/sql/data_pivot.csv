version https://git-lfs.github.com/spec/v1
oid sha256:458f0a8e8c1f8adcd6dd93d2d54ed249486b4a8dffd750d34f002c76092dbf22
size 38
