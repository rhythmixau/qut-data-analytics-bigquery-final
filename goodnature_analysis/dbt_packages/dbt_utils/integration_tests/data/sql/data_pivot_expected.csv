version https://git-lfs.github.com/spec/v1
oid sha256:3ea43003a6b86ef1ce3a5b0e6b6b7f66e822c5ae9320bd2ad1dbb58f209ac7bc
size 25
